app/views/hyrax/admin/collection_types/_form_settings.html.erb

```diff
@@ -3,11 +3,43 @@
 <p><strong><%= t('.warning') %></strong></p>
 
 <% disabled = (f.object.collections? || f.object.admin_set? || f.object.user_collection?) %>
+<% radio_disabled = (disabled || !f.object.sharable) %>
 <div class="collection-types-settings">
   <div class="form-inline">
     <p><%= f.input :nestable, as: :boolean, disabled: disabled %></p>
     <p><%= f.input :discoverable, as: :boolean, disabled: disabled %></p>
-    <p><%= f.input :sharable, as: :boolean, disabled: disabled %></p>
+    <p><%= f.input :sharable, as: :boolean, disabled: disabled %></p> <# TODO: need js to enable/disable radio buttons when this sharable is checked/unchecked %>
+    <p></p>
+    <div class="form-group collection_type_share_applies_to">
+      <div class="radio">
+        <label>
+          <%= f.radio_button :share_applies_to, :collection, checked: (f.object.share_applies_to==:share_applies_to_collection), disabled: radio_disabled %>
```

Comment:
This seems odd to me.  you are setting `share_applies_to = 'collection'` if it currently is `share_applies_to = 'share_applies_to_collection'`?  If you are using the form helpers correctly, you should never have to set `checked:` it should do it for you.